packages/legacy/core/App/navigators/RootStack.tsx

```diff
@@ -159,9 +156,40 @@ const RootStack: React.FC = () => {
     state.authentication.didAuthenticate,
   ])
 
+  useEffect(() => {
+    if (!inBackground || !agent) {
+      return
+    }
+
+    agent?.mediationRecipient
```

Comment:
Since we are checking to make sure agent is defined above, is the optional chaining still needed?